content/posts/Welcoming-Messege.en.md

```diff
@@ -3,12 +3,12 @@ title: "Welcome to ArabR"
 date: 2022-01-23T18:01:49+03:00
 draft: false
 author: hussain.en
-tags:
+tags: vision
```

Comment:
Tags need to be in quotation marks. Refer to [this](https://github.com/ArabR/ArabR-Website/blob/main/CONTRIBUTING.md#how-to-add-a-post) to learn more about the theme parameters

Reply:
Sorry I had not test your PR until now. The problem can be fixed by making tags a list. It should look something like this 
```
tags: 
     - "vision"
```
It seems that quotations are optional.